src/seedu/addressbook/data/person/Name.java

```diff
@@ -57,8 +58,13 @@ public boolean equals(Object other) {
     }
 
     @Override
-    public int hashCode() {
-        return fullName.hashCode();
+    public int hashCode() { return fullName.hashCode(); }
```

Comment:
Check the coding standard again, see if it allows you to write this code in this format or not